NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -197,33 +204,26 @@ def run(
         d_f = freqs[2] - freqs[1]
 
         if passband is None:
-            passband = [30 * units.MHz, 80 * units.MHz]
+            passband = [10 * units.MHz, 1000 * units.MHz]
+
         passband_filter = (freqs > passband[0]) & (freqs < passband[1])
 
         site_latitude, site_longitude = detector.get_site_coordinates(station.get_id())
-        site_location = astropy.coordinates.EarthLocation(lat=site_latitude * astropy.units.deg,
-                                                          lon=site_longitude * astropy.units.deg)
         station_time = station.get_station_time()
 
-        local_cs = astropy.coordinates.AltAz(location=site_location, obstime=station_time)
-        solid_angle = healpy.pixelfunc.nside2pixarea(self.__n_side, degrees=False)
-
-        pixel_longitudes, pixel_latitudes = healpy.pixelfunc.pix2ang(self.__n_side,
-                                                                     range(healpy.pixelfunc.nside2npix(self.__n_side)),
-                                                                     lonlat=True)
-        pixel_longitudes *= units.deg
-        pixel_latitudes *= units.deg
-
-        galactic_coordinates = astropy.coordinates.Galactic(l=pixel_longitudes * astropy.units.rad,
-                                                            b=pixel_latitudes * astropy.units.rad)
-        local_coordinates = galactic_coordinates.transform_to(local_cs)
+        local_coordinates = get_local_coordinates((site_latitude, site_longitude), station_time, self.__n_side)
 
         n_ice = ice.get_refractive_index(-0.01, detector.get_site(station.get_id()))
-        n_air = 1.000292  # TODO: This value applies under standard conditions. Does it hold for Greenland?
+        n_air = ice.get_refractive_index(depth=1, site=detector.get_site(station.get_id()))
```

Comment:
Isn't `ice` a deprecated class? If it also just return a hardcoded float I would prefer to define it here openly in the code.

Reply:
Better to use the (to be deprecated) class, IMO - we will catch all instances of this when we deprecate it.

Reply:
You do not mind to ask `ice` for `n_air`?

Reply:
also the argument `depth=1` is very confusing. I would assume that depth is positively defined and that depth = 1 is equal to z=-1. I hope this is not the case.

Reply:
What I'm worried about is having multiple hardcoded values for `n_air` in different places in the code base, which potentially differ in the last digit(s), which will probably somehow lead to some very hard to debug behaviour in some bizarre edge case far into the future. I am not going to defend the `ice` class which indeed should be deprecated.